### PR TITLE
Create TODO_Mainnet.md

### PR DESCRIPTION
**Pull request by tudor-malene**
### Why this change is needed

This document contains an overview of the outstanding work necessary for mainnet.
Once we're happy with it, these items should become epics and stories.

### CODE REVIEW

File: TODO_Mainnet.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Outstanding Work for Mainnet
```
[the remark this answers is not in the document]
Comment (tudor-malene) [reply TODO_Mainnet.md:1]: added to the security work

File: TODO_Mainnet.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  - review everything
+  - prepare docs for external security audit
+  - arrange the audit
+  - address the outcome of the audit
+
+## VII. Cross chain
+  - Sort out finality for xchain messages, separate from the DA rollup.
+
+## VIII. Management contract
+  -
```
Comment (otherview) [inline TODO_Mainnet.md:36]: Upgradable contracts
```suggestion
## VIII. Management contract
  - Upgradable contracts
  - Contract signature handling (multisig to deploy new versions ?)
```

**Comment by otherview** [comment]
@coderabbitai ignore

**Comment by otherview** [comment]
@coderabbitai pause

**Comment by tudor-malene** [comment]
this is now converted into epics

**Pull request closed without merging**